.github/workflows/build.yml

```diff
@@ -187,7 +187,10 @@ jobs:
       # This naming convention will be used ONLY for per-commit dev images
       - name: Set docker dev tag
         run: |
-          echo "dev_tag=${{ env.version }}" >> $GITHUB_ENV
+          echo "full_dev_tag=${{ env.version }}"
```

Comment:
**praise:** I like that you echo out the tags... it will help with debugging in the future.